### PR TITLE
Feature Accessibility

### PR DESCRIPTION
**Pull request by IsmailChangezi**
##Description
- Added the wireframe with Index.html
- Added functionality with JS
- Added the API 
- Fixed Linter Errors

### CODE REVIEW

File: dist/main.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+ * ATTENTION: The "eval" devtool has been used (maybe by default in mode: "development").
+ * This devtool is neither made for production nor for readable output files.
+ * It uses "eval()" calls to create a separate source file in the browser devtools.
+ * If you are trying to read the output file, select a different devtool (https://webpack.js.org/configuration/devtool/)
+ * or disable the default devtool with "devtool: false".
+ * If you are looking for production-ready output files, see mode: "production" (https://webpack.js.org/configuration/mode/).
+ */
+/******/ (() => { // webpackBootstrap
+/******/ 	"use strict";
+/******/ 	var __webpack_modules__ = ({
```
Comment (Munsa1) [inline dist/main.js]: - [ ] Kindly configure your webpack correctly, you're missing important files such as `package.json`, `webpack.config` e.t.c. Watch this short [tutorial](https://www.youtube.com/watch?v=HNb6bapmsyI)
- [ ] the following scripts must be present in your `package.json` for running and bundling the app
image

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        </div>
+        <div class="add">
+          <h2>add your score</h2>
+        </div>
+      </div>
+      <div class="score">
+        <ul class="scorelist"></ul>
+        <form action="" id="form-add">
+          <div>
+            <input type="text" name="name" placeholder="Your Name" />
```
Comment (Munsa1) [inline dist/index.html]: - [ ] Although styling is not required at this point, your wireframe needs to match the given wireframe.

Your design
image

Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @IsmailChangezi, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have webpack setup properly. Make sure everything works smoothly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**